coil-base/src/main/java/coil/ImageLoader.kt

```diff
@@ -52,9 +54,6 @@ import java.io.File
  *
  * Image loaders are designed to be shareable and work best when you create a single instance and
  * share it throughout your app.
- *
- * It's recommended, though not required, to call [shutdown] when you've finished using an image loader.
```

Comment:
Removed this and clarified that you probably don't need to shut down image loaders in the `shutdown` method doc.